input/config/ubuntu/22.04/intelgaudi.json

```diff
@@ -0,0 +1,51 @@
+{
+    "intelgaudi": {
+      "cluster": [
+      { 
+        "package": "habanalabs-dkms={{ intelgaudi_version }}",
+        "type": "deb",
+        "repo_name": "gaudi"
+      },
```

Comment:
different subgroups for Guadi drivers and Habana stack

Reply:
so Gaudi.jason should have subgroup details as below:
![image](https://github.com/user-attachments/assets/85ebc8d3-4eb4-4110-a37f-786b351e4857)

Reply:
will be taken care of in a subsequent PR